src/http/api/views/account.follows.view.ts

```diff
@@ -90,6 +137,80 @@ export class AccountFollowsView {
         };
     }
 
+    private async getRemoteFollowsPage(
+        actor: Actor,
+        next: URL,
+        documentLoader: DocumentLoader,
+    ): Promise<Result<CollectionPage, GetFollowsError>> {
+        let page: CollectionPage | null = null;
+
+        try {
+            // Ensure the next parameter is for the same host as the actor. We
+            // do this to prevent blindly passing URIs to lookupObject (i.e next
+            // param has been tampered with)
+            const { host: actorHost } = actor?.id || new URL('');
```

Comment:
`new URL('')` will always throw FYI - we have it in a `try/catch` block though so it's okay for now, but just a heads up